react/index.js

```diff
@@ -1,21 +1,21 @@
 // Components
-export { default as StyleGuideProvider } from './StyleGuideProvider/StyleGuideProvider';
-export { default as SeekApp } from './StyleGuideProvider/StyleGuideProvider';  // Alias for backwards compatibility
 export { default as Button } from './Button/Button';
-export { default as Logo } from './Logo/Logo';
-export { default as Rating } from './Rating/Rating';
-export { default as Header } from './Header/Header';
 export { default as Footer } from './Footer/Footer';
+export { default as Header } from './Header/Header';
 export { default as Loader } from './Loader/Loader';
+export { default as Logo } from './Logo/Logo';
+export { default as Rating } from './Rating/Rating';
+export { default as SeekApp } from './StyleGuideProvider/StyleGuideProvider';  // Alias for backwards compatibility
```

Comment:
We should break them out and place them at the very top:

```javascript
export { default as SeekApp } from './StyleGuideProvider/StyleGuideProvider'; 
export { default as StyleGuideProvider } from './StyleGuideProvider/StyleGuideProvider';

// Components
```
What do you think?